daemon/plugin_p.h

```diff
@@ -73,6 +74,8 @@ class PluginHelper : public QPluginLoader
             return 0;
         }
         qCDebug(category) << "loading plugin:" << fileName() << "with name:" << plugin->name();
+        qCDebug(category) << "initialising plugin:" << fileName();
+        plugin->initialise(parameters);
```

Comment:
Is this a place where real plugin (*.so file) is loaded?
If yes, then where is the result of initialize() check? it return true or false, then may be
```c
if (plugin->initialise(parameters))
...
else
...
```